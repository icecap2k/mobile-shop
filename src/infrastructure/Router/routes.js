import ProductList from "../../domain/ProductList.js";
import Product from "../../domain/Product";

export default [
  { path: "/", name: "Home", Component: ProductList },
  {
    path: "/products/:productId",
    name: "Product detail",
    Component: Product,
  },
];
