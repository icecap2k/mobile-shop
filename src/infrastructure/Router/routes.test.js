import routes from "./routes";

test("Routes are exported correctly", () => {
  expect(routes.map(({ name, path }) => `${name}: ${path}`)).toMatchSnapshot();
});
