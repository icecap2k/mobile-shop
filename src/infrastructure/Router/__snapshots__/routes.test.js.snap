// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes are exported correctly 1`] = `
Array [
  "Home: /",
  "Product detail: /products/:productId",
]
`;
